docker-compose.example.yml

```diff
@@ -2,12 +2,11 @@ version: '3'
 
 services:
   woodpecker-server:
-    build:
-      context: .
-      dockerfile: Dockerfile
-    image: woodpeckerci/woodpecker-server:local
+    image: woodpeckerci/woodpecker-server:latest
```

Comment:
Do we want to suggest latest here? (Sorry, I know somewhat off-topic but I just saw it here). I think it should be possible to also let this being updated dynamically via renovate without manual work by ourselve.

Reply:
Yes but that's exactly the point: you will also never know which version/commit you're actually on. If we suggest such examples, users will use it like this and never change it. Hence I suggest we always put the most recent version into the example.